ckan/controllers/group.py

```diff
@@ -88,8 +85,6 @@ def _bulk_process_template(self, group_type):
     ## end hooks
     def _replace_group_org(self, string):
         ''' substitute organization for group if this is an org'''
-        if self.group_type == 'organization':
-            string = re.sub('^group', 'organization', string)
```

Comment:
Only the OrganizationController has customized logic functions - it is not to do with the group_type. Therefore this code has been moved into OrganizationController.